setup.sh

```diff
@@ -70,8 +70,12 @@ function _check_root
 {
   if [[ ${EUID} -ne 0 ]]
   then
-    echo "Curently docker-mailserver doesn't support podman's rootless mode, please run this script as root user."
-    exit 1
+    read -p "You are now running in the rootless mode of podman, are you sure you want to continue? (Y/N):" -r yn
+    case ${yn} in
+      [Yy]* ) ;;
+      [Nn]* ) exit 1;;
+      * ) echo "Please answer Y(es) or N(o)."; exit 1;;
+    esac
```

Comment:
Please use this suggestion to follow general `read` conventions (chose a default value, make use of the `${REPLY}` variable, etc.). Make sure to adhere to the style conventions (always uppercase variables) too :)

``` SUGGESTION
    read -r -p "You are now running in the rootless mode of podman, are you sure you want to continue? [Y/n] "
    if [[ -n ${REPLY} ]] && [[ ${REPLY} =~ (n|N) ]]
      exit 0
   fi
```